asf_search/search/search_generator.py

```diff
@@ -272,16 +312,23 @@ def _get_product_type_key(item: Dict) -> str:
     collection_shortName = ASFProduct.umm_get(item['umm'], 'CollectionReference', 'ShortName')
 
     if collection_shortName is None:
-        platform_shortname = ASFProduct.umm_get(item['umm'], 'Platforms', 0, 'ShortName')
-        if platform_shortname in ['SENTINEL-1A', 'SENTINEL-1B']:
-            asf_platform = ASFProduct.umm_get(item['umm'], 'AdditionalAttributes', ('Name', 'ASF_PLATFORM'), 'Values', 0)
-            if 'Sentinel-1 Interferogram' in asf_platform:
-                return 'ARIA S1 GUNW'
+        platform = _get_platform(item=item)
+        if _is_ARIA_S1_GUNW(platform=platform, item=item):
+            return 'ARIA S1 GUNW'
 
-        return platform_shortname
+        return platform
 
     return collection_shortName
 
+def _get_platform(item: Dict):
+    return ASFProduct.umm_get(item['umm'], 'Platforms', 0, 'ShortName')
+
+def _is_ARIA_S1_GUNW(platform: str, item: Dict) -> bool:
+    if platform in ['SENTINEL-1A', 'SENTINEL-1B']:
+        asf_platform = ASFProduct.umm_get(item['umm'], 'AdditionalAttributes', ('Name', 'ASF_PLATFORM'), 'Values', 0)
+        return 'Sentinel-1 Interferogram' in asf_platform
+    
+    return False
```

Comment:
Is this the right spot for ARIA-specific stuff?

Reply:
Good point, moved to the class itself as a static method!